controller/src/test/java/org/jboss/as/controller/test/GlobalOperationsTestCase.java

```diff
@@ -954,6 +954,17 @@ public void testReadResourceAttributesOnly() throws Exception {
         executeForFailure(operation);
     }
 
+    @Test
```

Comment:
that's not what I suggested. your test is creating a resource with resolve-expressions set to `true`. This has no effect.

What you should do instead is something like:


```
    @Test
    public void testRecursiveReadSubModeWithResolveExpression() throws Exception {
        try {
            // TODO: set the system property expr to a random value
            String expr = ....
            System.setProperty("expr", expr);

            ModelNode operation = createOperation(READ_RESOURCE_OPERATION, "profile", "profileA");
            operation.get(RECURSIVE).set(true);
            operation.get(RESOLVE_EXPRESSIONS).set(true);

            ModelNode result = executeForResult(operation);
            assertNotNull(result);

           // check that the sub resource subsystem=subsystem2 has an attribute expression whose value correspond to the exp system property
           assertEquals(expr, result.get("subsystem", "subsystem2", "expression").asString());     
          } finally {
               // TODO unset the system property expr 
    }
```

Reply:
@jmesnil Thank you for reviewing it,I will take a look and fix the further quoted issues